src/components/App.tsx

```diff
@@ -36,6 +45,14 @@ function App(props: AppProps) {
   const [modalSpouse, setModalSpouse] = useState(null as Person | null);
   const [treeYaml, setTreeYaml] = useState('');
 
+  const { t, i18n } = useTranslation();
+  const [language, setLanguage] = useCache('language', 'en');
```

Comment:
Default language is the current i18n language
```suggestion
  const [language, setLanguage] = useCache('language', i18n.language);
```